client/post-editor/editor-html-toolbar/index.jsx

```diff
@@ -509,51 +509,49 @@ export class EditorHtmlToolbar extends Component {
 
 		return (
 			<div className={ insertContentClasses }>
-				<div
+				<button
```

Comment:
Let's save this refactor for a different PR

Reply:
This refactor is needed to clear the precommit a11y lint though, and it only requires 2 lines of CSS to look the same.

Reply:
ah gotcha 👍 I missed that in the summary.